phpstan.neon

```diff
@@ -4,6 +4,9 @@ parameters:
 		- src
 		- tests
 		- bdi
+	ignoreErrors:
+		- '#caught \"Throwable\" must be rethrown#'
```

Comment:
Hmm I don't want to ignore this one globally. You can add an ignore comment in the code where this applies (https://phpstan.org/user-guide/ignoring-errors)

Reply:
Ignore comment is available with phpstan 0.12+ and I am obliged to use phpstan 0.10 (because of packages `thecodingmachine/phpstan-safe-rule` and `thecodingmachine/safe`)
I updated the message to be ignored in order to be more generic. It is clearly mentioning now that only message about `\DBrekelmans\BrowserDriverInstaller\Command\DetectCommand::execute` method must be ignored